elevenlabs_mcp/server.py

```
@@ -1215,6 +1215,72 @@ def create_composition_plan(
+        tool_id: The ID of the tool to retrieve
+
+    Returns:
+        TextContent containing the raw JSON from the API
+    """
+    if tool_id == "":
+        make_error("Tool ID is required.")
+    try:
+        response = client.conversational_ai.tools.get(tool_id=tool_id)
+        return TextContent(type="text", text=response.model_dump_json(indent=2))
+
+    except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)